nodejs_mic_vad_streaming/Readme.md

```diff
@@ -5,40 +5,72 @@ DeepSpeech with voice activity detection.
 
 ### Prerequisites:
 
-1) The example utilized the [mic](https://github.com/ashishbajaj99/mic) NPM module which requires
-either [sox](http://sox.sourceforge.net/) (Windows/Mac) or [arecord](http://alsa-project.org/) (Linux).
+Node LTS v14.14 installed:
```

Comment:
That looks super-strict, can we have more details ? Is it what you tested to be successfull, or are there issues with other versions ?

Reply:
That's just what I tested with, happy to change it to just 'tested with v14.14'. Not sure what versions it runs nicely with unfortunately and don't feel like testing all the versions 😅